php/libraries/Utility.class.inc

```diff
@@ -150,6 +151,39 @@ class Utility
         return $list;
     }
 
+    /**
+     * Returns a list of sites by site IDs from the database
+     *
+     * @param array $siteIDs a list of the study center IDs
+     *
+     * @return array an associative array("center ID" => "site name")
+     */
+    static function getSiteListByIDs(array $siteIDs): array
```

Comment:
I would rename the function to `getSiteNameListFromIDs()` because you are not getting sites but site names.

Reply:
`getSiteListByIDs` is to be consistent with `getSiteList`

Reply:
`getsitelist` is incorrectly named because we were less rigorous before, since it returns names not sites I think it should be properly named